Tasks/AzureNLBManagement/nlbUtility.ts

```diff
@@ -0,0 +1,194 @@
+import * as tl from 'vsts-task-lib/task';
+import * as Q from 'q';
+import * as querystring from "querystring";
+import * as httpClient from 'vso-node-api/HttpClient';
+import * as restClient from 'vso-node-api/RestClient';
+
+var httpObj = new httpClient.HttpCallbackClient(tl.getVariable("AZURE_HTTP_USER_AGENT"));
+var restObj = new restClient.RestCallbackClient(httpObj);
+
+var authUrl = 'https://login.windows.net/';
+var azureApiVersion = '2016-09-01';
+
+export function getAccessToken(SPN, endpointUrl: string): Q.Promise<string> {
```

Comment:
I see this as a common function. Should this be refactored as independent utility that other task can use? @mvvsubbu - Thoughts?